package.json

```diff
@@ -17,7 +17,9 @@
         "lint": "npx tslint -c tslint.json --project . && npx prettier --list-different './packages/**/src/*.ts'"
     },
     "devDependencies": {
+        "@cennznet/cennznut": "^1.0.0",
```

Comment:
the root package.json is for common dev dependencies, mostly non-bussness-related.

Reply:
What's the proper command to install a dev dependency for this project? I think I did `yarn add <package> -D`